ui/helpers/utils/conversions.util.js

```diff
@@ -7,6 +7,16 @@ import {
 } from '../../../shared/modules/conversion.utils';
 import { formatCurrency } from './confirm-tx.util';
 
+export function toHex(value) {
+  let result = value;
+  if (!result) return result;
+  if (result !== typeof 'string') {
```

Comment:
Should this be `if (typeof result !== 'string')`?

Reply:
This method was used in a previous iteration, and is no longer needed! Will remove.

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13048/commits/e0a1b4c80da158cc529475d6cdf7faca39d269d6